feature/feature.go

```diff
@@ -38,6 +38,11 @@ const (
 	//
 	// alpha: v0.3
```

Comment:
```suggestion
	// alpha: v0.3
	// beta: v0.4
```
Just in case you want to fix that / I'm even right :)